apps/docs/package.json

```diff
@@ -6,7 +6,8 @@
   "eslintConfig": {
     "extends": [
       "@repo/eslint-config"
-    ]
+    ],
+    "ignorePatterns": ["out/**/*", ".next/**/*"]
```

Comment:
Node modules maybe?

Reply:
Although why ignore these if you've changed the linting to run on the src folder?

Reply:
Src folder was only for `create-guider` project

Reply:
Also node_modules are ignore by default by eslint